proto/src/determined/api/v1/api.proto

```diff
@@ -2687,7 +2675,7 @@ service Determined {
     };
   }
 
-  // Kill runs.
+  // Get a list of runs.
```

Comment:
Typo? We have `/api/v1/runs` for getting a list of runs.

Reply:
I'll update this comment.